index.php

```diff
@@ -2,9 +2,17 @@
 require_once 'produkt.php';
 
 
-$jablko = new Produkt('jablko', '5');
-$bombonierka = new Produkt('bombonierka','10');
+    $produkt[1] = [ new Produkt('jablko', '5')];
```

Comment:
Jeżeli tworzysz tablicę z liczbami jako klucz to możesz to zrobić tak: `$produkt = ['element', 'element']`

Reply:
Po drugie, po co tworzysz tablicę, która zawiera tablice (`[ new Produkt() ]` zamiast `new Produkt`? Z tego wynika błąd, który dostajesz. Zrób sobie var_dumpa $wartosc i zobaczysz, że dostaniesz tablicę (na której nie możesz wywołać getNazwa) zamiast obiektu (na którym możesz).